README.md

```diff
@@ -9,6 +9,7 @@ mpv-android is a video player for Android based on [libmpv](https://github.com/m
 * Hardware and software video decoding
 * Gesture-based seeking, volume/brightness control and more
 * libass support for styled subtitles
+* Secondary (or dual) subtitle support
```

Comment:
We can wait but then this PR will be left open for the next 6 months.

Reply:
No problem, I hope sooner of course. Perhaps trivial, but this feature of dual subtitles is really something MPV may boost about / position itself with. It is very useful for learning languages. VLC supports it in 4.0, which is still in beta, and Kodi has some community hack for it online (merging SRT files) but is far from supporting it.

Off topic: In case this is relevant for MPV, F-Droid also supports dev/beta releases such as https://f-droid.org/en/packages/org.openhab.habdroid.beta/ and https://f-droid.org/en/packages/com.nextcloud.android.beta/